cli/cmd/install.go

```diff
@@ -418,12 +418,14 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 // during upgrade.  Note also that this Secret/linkerd-config-overrides
 // resource is not part of the Helm chart and will not be present when installing
 // with Helm.
-func renderOverrides(values *l5dcharts.Values, namespace string) ([]byte, error) {
+func renderOverrides(values chartutil.Values, namespace string) ([]byte, error) {
```

Comment:
Why not simply using `controlPlaneNamespace` and forgo the `namespace` arg?

Reply:
We can't do that as we would miss it if the namespace flag is overwritten through `global.namespace` helm field.

Reply:
Gotcha :+1: